task-definition.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  default_container_definitions_used = var.container_definitions == "" ? true : false
```

Comment:
`var.container_definitions` doesn't really tell me anything. Can we make its default a null, and check against a null here too?